build(aio): update to @angular/core@5.0.0-rc.5

IgorMinar opened the pull request:
there is a size regression right now because the CLI is out of date:

-rw-r--r--  1 iminar  eng   14942 Oct 13 10:23 dist/0.b19e913fbdd6507d346b.chunk.js
-rw-r--r--  1 iminar  eng    1535 Oct 13 10:23 dist/inline.812a4af83ecce165c71c.bundle.js
-rw-r--r--  1 iminar  eng  643481 Oct 13 10:23 dist/main.74550bb35f9f5a57e78a.bundle.js
-rw-r--r--  1 iminar  eng   37402 Oct 13 10:23 dist/polyfills.965a9a5ad3e11b17f85e.bundle.js

IgorMinar committed:
build(aio): update to @angular/core@5.0.0-rc.2

there is a size regression right now because the CLI is out of date:

-rw-r--r--  1 iminar  eng   14942 Oct 13 10:23 dist/0.b19e913fbdd6507d346b.chunk.js
-rw-r--r--  1 iminar  eng    1535 Oct 13 10:23 dist/inline.812a4af83ecce165c71c.bundle.js
-rw-r--r--  1 iminar  eng  643481 Oct 13 10:23 dist/main.74550bb35f9f5a57e78a.bundle.js
-rw-r--r--  1 iminar  eng   37402 Oct 13 10:23 dist/polyfills.965a9a5ad3e11b17f85e.bundle.js

IgorMinar committed:
build(aio): upgrade to @angular/cli@1.4.7

this causes the size regression to get only worse:

-rw-r--r--  1 iminar  eng   14942 Oct 13 10:37 dist/0.b19e913fbdd6507d346b.chunk.js
-rw-r--r--  1 iminar  eng    1535 Oct 13 10:37 dist/inline.3cc964095cb25e329dc0.bundle.js
-rw-r--r--  1 iminar  eng  846141 Oct 13 10:37 dist/main.5eb64df77b2877327a16.bundle.js
-rw-r--r--  1 iminar  eng   37402 Oct 13 10:37 dist/polyfills.965a9a5ad3e11b17f85e.bundle.js

IgorMinar committed:
build(aio): upgrade to @angular/cli@1.5.0-rc.0 and typescript@2.5.3

the size regression has gotten worse:

-rw-r--r--  1 iminar  eng   14942 Oct 13 12:24 dist/0.b19e913fbdd6507d346b.chunk.js
-rw-r--r--  1 iminar  eng    1535 Oct 13 12:24 dist/inline.41e701c562960ede8ef5.bundle.js
-rw-r--r--  1 iminar  eng  865780 Oct 13 12:24 dist/main.6c4c605d461870b9c2d7.bundle.js
-rw-r--r--  1 iminar  eng   37402 Oct 13 12:24 dist/polyfills.f8409a9eb69060ac1aa6.bundle.js

IgorMinar committed:
build(aio): upgrade to @angular/{material,cdk}@2.0.0-beta.11 + md->mat migration

all the non-npm changes were made by the angular-material-prefix-updater tool.

the tool missed a few things, which I'll fix in a separate commit to preserve the diff.

-rw-r--r--  1 iminar  eng   14942 Oct 13 13:09 dist/0.b19e913fbdd6507d346b.chunk.js
-rw-r--r--  1 iminar  eng    1535 Oct 13 13:09 dist/inline.0592c25ceb544d6aca3d.bundle.js
-rw-r--r--  1 iminar  eng  578250 Oct 13 13:09 dist/main.45d4edca3facc6d621e7.bundle.js
-rw-r--r--  1 iminar  eng   37402 Oct 13 13:09 dist/polyfills.f8409a9eb69060ac1aa6.bundle.js

IgorMinar committed:
fix(aio): hand fix the renaming md->mat issues

These are changes that the mat-switcher missed and I had to make them by hand.

IgorMinar committed:
build(aio): upgrade to @angular/{material,cdk}@2.0.0-beta.12

-rw-r--r--  1 iminar  eng   14942 Oct 13 13:35 dist/0.b19e913fbdd6507d346b.chunk.js
-rw-r--r--  1 iminar  eng    1535 Oct 13 13:35 dist/inline.f005f1bd6803b72f5961.bundle.js
-rw-r--r--  1 iminar  eng  582527 Oct 13 13:35 dist/main.b9ef1abb785be8de15b8.bundle.js
-rw-r--r--  1 iminar  eng   37402 Oct 13 13:35 dist/polyfills.f8409a9eb69060ac1aa6.bundle.js

IgorMinar committed:
build(aio): upgrade to rxjs@5.5.0-beta.7

-rw-r--r--  1 iminar  eng   14942 Oct 13 14:30 dist/0.b19e913fbdd6507d346b.chunk.js
-rw-r--r--  1 iminar  eng    1535 Oct 13 14:30 dist/inline.702d6ff5146ddc373f05.bundle.js
-rw-r--r--  1 iminar  eng  588943 Oct 13 14:30 dist/main.64c96d55a10c56cfd6cd.bundle.js
-rw-r--r--  1 iminar  eng   37402 Oct 13 14:30 dist/polyfills.f8409a9eb69060ac1aa6.bundle.js

IgorMinar committed:
build(aio): patch @angular/cli to use esm builds of rxjs

-rw-r--r--  1 iminar  eng   14942 Oct 13 14:52 dist/0.b19e913fbdd6507d346b.chunk.js
-rw-r--r--  1 iminar  eng    1535 Oct 13 14:52 dist/inline.6ca24f1c3b848103b041.bundle.js
-rw-r--r--  1 iminar  eng  567802 Oct 13 14:52 dist/main.c8183a2c0116782ca366.bundle.js
-rw-r--r--  1 iminar  eng   37402 Oct 13 14:52 dist/polyfills.f8409a9eb69060ac1aa6.bundle.js

IgorMinar committed:
build(aio): turn off preserveWhitespaces in compiler options

-rw-r--r--  1 iminar  eng   14942 Oct 13 16:12 dist/0.b19e913fbdd6507d346b.chunk.js
-rw-r--r--  1 iminar  eng    1535 Oct 13 16:12 dist/inline.eede8140efeab4c45b22.bundle.js
-rw-r--r--  1 iminar  eng  559389 Oct 13 16:12 dist/main.20858f9aa7cf8741b6aa.bundle.js
-rw-r--r--  1 iminar  eng   37402 Oct 13 16:12 dist/polyfills.f8409a9eb69060ac1aa6.bundle.js

mary-poppins commented:
You can preview f9397f9 at https://pr19702-f9397f9.ngbuilds.io/.

IgorMinar committed:
build(aio): upgrade to rxjs@5.5.0

-rw-r--r--  1 iminar  eng   84219 Oct 18 09:13 dist/0.8ef208c27531d5c6af63.chunk.js
-rw-r--r--  1 iminar  eng   14942 Oct 18 09:13 dist/4.c719ac5645940382cdce.chunk.js
-rw-r--r--  1 iminar  eng    1560 Oct 18 09:13 dist/inline.adc367eb50c706f3fd04.bundle.js
-rw-r--r--  1 iminar  eng  492354 Oct 18 09:13 dist/main.b9d9549455c74aff1480.bundle.js
-rw-r--r--  1 iminar  eng   37402 Oct 18 09:13 dist/polyfills.f8409a9eb69060ac1aa6.bundle.js

gkalpak commented on aio/package.json:
```diff
@@ -82,7 +82,7 @@
     "jasmine": "^2.6.0",
     "ng-pwa-tools": "^0.0.10",
-    "rxjs": "^5.2.0",
+    "rxjs": "5.5.0-beta.7",
```
We could get to `5.5.0` now :grin:

gkalpak commented on aio/tools/cli-patches/patch.js:
```diff
@@ -0,0 +1,10 @@
+const fs = require('fs');
+const sh = require('shelljs');
+
+PATCH_LOCK = 'node_modules/@angular/cli/models/webpack-configs/.patched';
```
_`PATCH_LOCK`_ --> _`const PATCH_LOCK`_(?)

gkalpak commented on aio/tools/cli-patches/rxjs-esm-aliases.patch:
```diff
@@ -0,0 +1,11 @@
+             extensions: ['.ts', '.js'],
+             modules: ['node_modules', nodeModules],
+             symlinks: !buildOptions.preserveSymlinks
++           ,alias: require('rxjs/_esm5/path-mapping')(path.resolve(__dirname, '..', '..', '..', '..', '..', 'node_modules'))
```
`path.resolve(__dirname, '../../../../../node_modules')` woks too :grin:

gkalpak commented on aio/tools/cli-patches/patch.js:
```diff
@@ -0,0 +1,10 @@
+PATCH_LOCK = 'node_modules/@angular/cli/models/webpack-configs/.patched';
+
+if (!fs.existsSync(PATCH_LOCK)) {
+  sh.exec('patch -p0 -i tools/cli-patches/rxjs-esm-aliases.patch');
```
Nit: You might want to use `sh.set('-e')`, so that if an errors occurs the `PATCH_LOCK` file isn't touched.

gkalpak commented on aio/src/app/app.module.ts:
```diff
@@ -45,10 +35,9 @@ import { ScrollService } from 'app/shared/scroll.service';
 
-// These are the hardcoded inline svg sources to be used by the `<md-icon>` component
+// These are the hardcoded inline svg sources to be used by the `<mat-icon>` component
 export const svgIconProviders = [
```
It seems that these icons are not picked up any more:
![aio-missing-icons](https://user-images.githubusercontent.com/8604205/31743443-db0dc834-b463-11e7-9254-c51dce575113.png)

(Or is it related to the last commit ("deferring the embedded components"), which as discussed will be removed from this PR?)

IgorMinar replied:
@gkalpak would you have some extra cycles to help investigate why this is broken? I assume that it has to be related with our custom icon registry.

gkalpak replied:
Yeah, I will look into it.

gkalpak replied:
I verified this is indeed related to the commit that defers loading the embdded components. 
So, let's deal with this issue in another PR. (As discussed, deferring embedded components will be removed from this PR. This is still the plan, right?)

IgorMinar replied:
I removed the commit that caused this.

gkalpak commented on aio/package.json:
```diff
@@ -82,13 +82,13 @@
   },
   "devDependencies": {
-    "@angular/cli": "^1.3.0",
+    "@angular/cli": "^1.5.0-rc.0",
```
We could get to `rc.1` now :grin:

gkalpak commented on aio/src/app/app.module.ts:
```diff
@@ -1,36 +1,26 @@
-// crashes with "missing first" operator when SideNav.mode is "over"
-import 'rxjs/add/operator/first';
+import { ROUTES } from '@angular/router';
+import { Platform } from '@angular/cdk/platform';
```
Why not import it from `@angular/material` or `@angular/material/platform`?
Or better yet, import `PlatformModule` from `@angular/material[/platform]`?
(Assuming `Platform` is still necessary in the first place :smiley:)

IgorMinar replied:
I believe that export was deprecated.

mary-poppins commented:
You can preview 5e4944f at https://pr19702-5e4944f.ngbuilds.io/.

IgorMinar committed:
build(aio): upgrade to @angular/cli@1.5.0-rc.1

-rw-r--r--  1 iminar  eng   84219 Oct 18 21:05 dist/0.0f327734d18211139822.chunk.js
-rw-r--r--  1 iminar  eng   14942 Oct 18 21:05 dist/4.c719ac5645940382cdce.chunk.js
-rw-r--r--  1 iminar  eng    1560 Oct 18 21:05 dist/inline.887757679ff553e20b54.bundle.js
-rw-r--r--  1 iminar  eng  492354 Oct 18 21:05 dist/main.5e5dc9ed980c9f5dc2bd.bundle.js
-rw-r--r--  1 iminar  eng   37402 Oct 18 21:05 dist/polyfills.f8409a9eb69060ac1aa6.bundle.js

IgorMinar commented:
sounds good. let's defer the deferring

On Thu, Oct 19, 2017 at 3:26 AM George Kalpakas <notifications@github.com>
wrote:

> *@gkalpak* commented on this pull request.
> ------------------------------
>
> In aio/src/app/app.module.ts
> <https://github.com/angular/angular/pull/19702#discussion_r145660423>:
>
> >  import { SharedModule } from 'app/shared/shared.module';
>
> -// These are the hardcoded inline svg sources to be used by the `<md-icon>` component
> +// These are the hardcoded inline svg sources to be used by the `<mat-icon>` component
>  export const svgIconProviders = [
>
> I verified this is indeed related to the commit that defers loading the
> embdded components.
> So, let's deal with this issue in another PR. (As discussed, deferring
> embedded components will be removed from this PR. This is still the plan,
> right?)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/angular/angular/pull/19702#discussion_r145660423>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AANM6Mjmkrn8_bbe1o0ot9ud6oZQJEUGks5styPegaJpZM4P4zqI>
> .
>

IgorMinar committed:
build(aio): upgrade to @angular/cli@1.5.0-rc.2

-rw-r--r--  1 iminar  eng   84219 Oct 19 21:37 dist/0.0f327734d18211139822.chunk.js
-rw-r--r--  1 iminar  eng   14942 Oct 19 21:37 dist/4.c719ac5645940382cdce.chunk.js
-rw-r--r--  1 iminar  eng    1560 Oct 19 21:37 dist/inline.887757679ff553e20b54.bundle.js
-rw-r--r--  1 iminar  eng  492354 Oct 19 21:37 dist/main.5e5dc9ed980c9f5dc2bd.bundle.js
-rw-r--r--  1 iminar  eng   37402 Oct 19 21:37 dist/polyfills.f8409a9eb69060ac1aa6.bundle.js

IgorMinar committed:
build(aio): upgrade to build-optimizer@0.0.29

-rw-r--r--  1 iminar  eng   14942 Oct 20 22:16 dist/0.b19e913fbdd6507d346b.chunk.js
-rw-r--r--  1 iminar  eng    1535 Oct 20 22:16 dist/inline.68ebcf831dc9c905804f.bundle.js
-rw-r--r--  1 iminar  eng  541291 Oct 20 22:16 dist/main.5ec6fb5f95fc0433d822.bundle.js
-rw-r--r--  1 iminar  eng   37402 Oct 20 22:16 dist/polyfills.f8409a9eb69060ac1aa6.bundle.js

IgorMinar committed:
build(aio): disable 'global' support in webpack

This will be fixed in CLI once https://github.com/angular/angular-cli/pull/8130 lands.

-rw-r--r--  1 iminar  eng   14942 Oct 20 22:23 dist/0.b19e913fbdd6507d346b.chunk.js
-rw-r--r--  1 iminar  eng    1535 Oct 20 22:23 dist/inline.5d66b81ec9e01af9d28d.bundle.js
-rw-r--r--  1 iminar  eng  528395 Oct 20 22:23 dist/main.e36bb99245ca52ae546f.bundle.js
-rw-r--r--  1 iminar  eng   37205 Oct 20 22:23 dist/polyfills.0dfca732c5a075c110d0.bundle.js

IgorMinar committed:
build(aio): temporarily increaze the size limit until the regressions are fixed

related issues:
https://github.com/angular/angular/issues/19857
https://github.com/angular/devkit/pull/231

IgorMinar commented:
I think that we should land this and get the fixes for #19857
angular/devkit#231 in a new PR once they are available.

mary-poppins commented:
You can preview f55a6a8 at https://pr19702-f55a6a8.ngbuilds.io/.

gkalpak commented on aio/tools/cli-patches/patch.js:
```diff
@@ -0,0 +1,10 @@
+const PATCH_LOCK = 'node_modules/@angular/cli/models/webpack-configs/.patched';
+
+if (!fs.existsSync(PATCH_LOCK)) {
+  sh.exec('patch -p0 -i tools/cli-patches/webpack-no-global.patch');
```
Nit: You might want to use `sh.set('-e')`, so that if an errors occurs the `PATCH_LOCK` file isn't touched.

gkalpak commented on aio/src/app/app.module.ts:
```diff
@@ -107,7 +111,7 @@ export const svgIconProviders = [
     { provide: LocationStrategy, useClass: PathLocationStrategy },
     LocationService,
-    { provide: MdIconRegistry, useClass: CustomMdIconRegistry },
+    { provide: MatIconRegistry, useClass: CustomMdIconRegistry },
```
Nit: Ideally, this should be renamed to `CustomMatIconRegistry` 😇

petebacondarwin replied:
In my PR, I renamed this to `CustomIconRegistry` to avoid any future name changes :-P

mary-poppins commented:
You can preview 9e1108e at https://pr19702-9e1108e.ngbuilds.io/.

gkalpak reviewed (approved):
LGTM (as soon as Travis is happy - unless you strongly feel like addressing my nits :grin:).

petebacondarwin commented:
Travis was not happy. I have restarted it. The error was TypeScript compilation in a Karma test failed!

petebacondarwin commented:
Still failing... https://travis-ci.org/angular/angular/jobs/291544530#L1353
I'll pull it down locally and debug...

petebacondarwin committed:
build(aio): revert to clean CLI test.ts file

The use of `System.import()` in test.ts was causing the webpack build to fail
with a mysterious "Module build failed: Error: TypeScript compilation failed" error,
when running `yarn test`.

mary-poppins commented:
You can preview 25c1ece at https://pr19702-25c1ece.ngbuilds.io/.

petebacondarwin commented:
Great! The test build is now working... but there are now 15 failing tests!
https://travis-ci.org/angular/angular/jobs/291761428

IgorMinar commented:
Sigh..

On Mon, Oct 23, 2017, 2:25 PM Pete Bacon Darwin <notifications@github.com>
wrote:

> Great! The test build is now working... but there are now 15 failing tests!
> https://travis-ci.org/angular/angular/jobs/291761428
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/angular/angular/pull/19702#issuecomment-338801389>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AANM6N51WN6pbJyXHj2tLlgLn_NX5v4Nks5svQQ7gaJpZM4P4zqI>
> .
>

petebacondarwin commented:
Progress...

petebacondarwin commented:
One step forward another step back...
![screen shot 2017-10-23 at 22 53 07](https://user-images.githubusercontent.com/15655/31915069-0661bfd8-b845-11e7-8d4a-2f07f79717e3.png)

I'll continue tomorrow.

petebacondarwin committed:
build(aio): remove cli patches

petebacondarwin committed:
build(aio): fix tests to work with @angular/{material,cdk}@2.0.0-beta.12

petebacondarwin committed:
build(aio): revert temporary increase in size limit

petebacondarwin committed:
build(aio): upgrade to Angular@rc.5 and CLI@rc.3

gkalpak commented:
This also needs to be taken care of: https://github.com/angular/angular/pull/19836#issuecomment-338934405

petebacondarwin committed:
build(aio): remove hack to modify CLI version

petebacondarwin commented:
Getting there....

petebacondarwin committed:
ci(aio): increase payload size limit for polyfills.ts

The latest builds have added ~2kB to the size of this file (500 bytes zipped).

mary-poppins commented:
You can preview d8516cb at https://pr19702-d8516cb.ngbuilds.io/.

petebacondarwin committed:
style(aio): fix docs linting issues

These issues appeared after upgrade of eslint jasmine plugin

mary-poppins commented:
You can preview 89783e1 at https://pr19702-89783e1.ngbuilds.io/.

IgorMinar commented on aio/scripts/_payload-limits.sh:
```diff
@@ -5,10 +5,10 @@ set -u -e -o pipefail
 payloadLimits["aio", "uncompressed", "inline"]=1600
 payloadLimits["aio", "uncompressed", "main"]=525500
-payloadLimits["aio", "uncompressed", "polyfills"]=38000
+payloadLimits["aio", "uncompressed", "polyfills"]=40100
```
why did the size increase? have we changed the zone.js version? that's the only polyfill we have.

please document the exact reason in the commit message (if you know it).

JiaLiPassion replied:
@IgorMinar , maybe the size increase because of this https://github.com/angular/zone.js/pull/916, with new version of typescript, the compiled `zone.js` will generate additional code for `iterable` and `spread operator`, this has been modified, so the size will be reduced but `zone.js` new version have not been released. (now is 0.8.18)

petebacondarwin committed:
build(aio): lock zone.js to 0.8.16

Later versions (before 0.8.19) had a size increase.

mary-poppins commented:
You can preview ed7cf6e at https://pr19702-ed7cf6e.ngbuilds.io/.

petebacondarwin commented:
I reverted the update to zone.js (by locking it to 0.8.16) and this allows us to revert the payload size increase for polyfills.ts. The build is now green on Travis. The CircleCI failures are due to the bad commit message during the recent 5.x release.

The AIO e2e tests are being a bit flaky, but I don't believe that is related to this PR.
If @IgorMinar is happy then we can squash and label for merge.

gkalpak commented on aio/tools/transforms/.eslintrc.js:
```diff
@@ -15,6 +15,7 @@ module.exports = {
     'quotes': ['error', 'single'],
-    'semi': ['error', 'always']
+    'semi': ['error', 'always'],
+    'jasmine/new-line-before-expect': 0
```
:100:

gkalpak commented on aio/tools/transforms/authors-package/index.spec.js:
```diff
@@ -1,3 +1,4 @@
+/* eslint jasmine/prefer-toHaveBeenCalledWith:0 */
```
:smiley:

IgorMinar commented:
Let's do the following:

- [ ] rebase
- [ ] decrease the size limits to prevent future regressions (we are now way below 520k)
- [ ] don't squash there is a lot of useful info in various commit messages which might be helpful in the future

petebacondarwin committed:
refactor(aio): rename `CustomMdIconRegistry` to `CustomIconRegistry`

The change of Angular Material version means that the `md` prefix is
no longer appropriate.

petebacondarwin commented:
Rebasing etc now...

petebacondarwin committed:
ci(aio): decrease payload size limit for main file

petebacondarwin commented:
Rebased.

mary-poppins commented:
You can preview 768f496 at https://pr19702-768f496.ngbuilds.io/.

IgorMinar commented on aio/scripts/_payload-limits.sh:
```diff
@@ -4,7 +4,7 @@ set -u -e -o pipefail
 declare -A payloadLimits
 payloadLimits["aio", "uncompressed", "inline"]=1600
-payloadLimits["aio", "uncompressed", "main"]=525500
+payloadLimits["aio", "uncompressed", "main"]=522500
```
Is this right? I was getting 490k last week with the latest build optimizer.

petebacondarwin replied:
I built it just now on this branch and it was 522215. Perhaps I am missing something?

petebacondarwin replied:
It seems that I was :-)
@IgorMinar spotted that we needed to update to the latest build-optimizer.

IgorMinar committed:
build(aio): upgrade to @angular/cli@1.5.0-rc.6

-rw-r--r--  1 iminar  eng   14880 Oct 30 11:29 dist/0.b19e913fbdd6507d346b.chunk.js
-rw-r--r--  1 iminar  eng    1533 Oct 30 11:29 dist/inline.25600c3b48de18b97581.bundle.js
-rw-r--r--  1 iminar  eng  486476 Oct 30 11:29 dist/main.d1292a34401056535884.bundle.js
-rw-r--r--  1 iminar  eng   37070 Oct 30 11:29 dist/polyfills.0dfca732c5a075c110d0.bundle.js

mary-poppins commented:
You can preview 3187c9f at https://pr19702-3187c9f.ngbuilds.io/.

petebacondarwin commented on aio/yarn.lock:
```diff
@@ -8401,7 +8408,7 @@ typescript@2.4:
   resolved "https://registry.yarnpkg.com/typescript/-/typescript-2.4.2.tgz#f8395f85d459276067c988aa41837a8f82870844"
 
-"typescript@>=2.0.0 <2.6.0", typescript@^2.3.3, typescript@^2.5.3:
+typescript@^2.3.3, typescript@^2.5.3:
```
+1

IgorMinar committed:
build(aio): upgrade to @angular/cli@1.5.0-rc.8

-rw-r--r--  1 iminar  eng   14880 Nov  1 11:57 dist/0.b19e913fbdd6507d346b.chunk.js
-rw-r--r--  1 iminar  eng    1533 Nov  1 11:57 dist/inline.3574d1d784c09c507dbd.bundle.js
-rw-r--r--  1 iminar  eng  497812 Nov  1 11:57 dist/main.76bbb69df79eaefef54c.bundle.js
-rw-r--r--  1 iminar  eng   37259 Nov  1 11:57 dist/polyfills.fdb71956ccd13330fb47.bundle.js

IgorMinar committed:
revert: build(aio): remove cli patches

This reverts commit f0d530b4de38f71c759e42afc8f3d7531eb1b1fb.

cli rc.8 reintroduces the node polyfill which causes size regression,
so I'm putting the patch back in.

-rw-r--r--  1 iminar  eng   14880 Nov  1 12:11 dist/0.b19e913fbdd6507d346b.chunk.js
-rw-r--r--  1 iminar  eng    1533 Nov  1 12:11 dist/inline.25600c3b48de18b97581.bundle.js
-rw-r--r--  1 iminar  eng  486476 Nov  1 12:11 dist/main.d1292a34401056535884.bundle.js
-rw-r--r--  1 iminar  eng   37070 Nov  1 12:11 dist/polyfills.0dfca732c5a075c110d0.bundle.js

IgorMinar committed:
build(aio): upgrade to @angular/core@5.0.0-rc.9

build fails - material upgrade required

IgorMinar committed:
build(aio): upgrade to @angular/material@2.0.0-beta.12

-rw-r--r--  1 iminar  eng   14880 Nov  1 12:25 dist/0.b19e913fbdd6507d346b.chunk.js
-rw-r--r--  1 iminar  eng    1533 Nov  1 12:25 dist/inline.2826385ad3e299c6d1c1.bundle.js
-rw-r--r--  1 iminar  eng  486476 Nov  1 12:25 dist/main.f0610805f4aad19da4be.bundle.js
-rw-r--r--  1 iminar  eng   37070 Nov  1 12:25 dist/polyfills.0dfca732c5a075c110d0.bundle.js

IgorMinar commented:
I rebased this again and upgraded all the dependencies again to latest rcs

mary-poppins commented:
You can preview 173201b at https://pr19702-173201b.ngbuilds.io/.

petebacondarwin commented:
Still approved and marked as merge :-)

mary-poppins commented:
You can preview db98207 at https://pr19702-db98207.ngbuilds.io/.

mary-poppins commented:
You can preview 1557319 at https://pr19702-1557319.ngbuilds.io/.

petebacondarwin commented:
Please merge me !!
